hardware/xilinx/scripts/vivado.tcl

```diff
@@ -80,6 +82,11 @@ set store_ip "${ip_path}/vta_store/soln/impl/ip/xilinx_com_hls_store_1_0.zip"
 # Create custom project
 create_project -force $proj_name $proj_path -part $device
 
+# Apply board preset if exists
+if {$board != "None" && $board_rev != "None"} {
+  set_property BOARD_PART $board:$board_rev [current_project]
```

Comment:
Just curious, why is BOARD_PART required here?

Reply:
When you build a hardware design Vivado let's you specify something called presets based on BOARD_PART. Presets are board specific configurations related to the hardware. Most importantly the DDR configuration. If you look at the hardware design built without a preset you will notice that the DDR defaults to 1600MHz. If you apply the preset (ZCU104 for this example) the DDR clock will be 2133MHZ.

If we want meaningful output products then we should specify this for the boards.

Reply:
AVNET has board files that can be used as external sources. Would be a good update for the Ultra96 hardware. 

https://github.com/Avnet/bdf/tree/master/ultra96v2/1.1

Reply:
@liangfu Let me know if you have any more questions. Would love to get this merged so I can submit the next PR to support the ultra96 variants. (v1 and v2 have different parts from a tools point of view - v2 is industrial grade).